libraries/botbuilder-ai/src/luis-client/luisClient.ts

```diff
@@ -202,6 +202,7 @@ export class LuisClient {
     public async predictionResolveGet(appId: string, query: string, options: PredictionResolveOptionalParams): Promise<LuisResult> {
         const localVarPath = this.basePath + '/apps/{appId}'
             .replace('{' + 'appId' + '}', encodeURIComponent(String(appId)));
+        var HttpStatus = require('http-status-codes');
```

Comment:
This is not OK. The _require/imports_ must be performed at the beginning of the file. You could implement something like this (check the third line):
![image](https://user-images.githubusercontent.com/43762887/62728499-5d4c0c80-b9f2-11e9-929a-332bd25432c0.png)

Reply:
Allright, I implemented your suggestion in the third line. Works good.